README.md

```diff
@@ -89,9 +116,20 @@ The following environment variables tweak DXVK-NVAPI's runtime behavior:
 - `DXVK_NVAPI_LOG_LEVEL` set to `info` prints log statements. The default behavior omits any logging. Please fill an issue if using log servery `info` creates log spam. Setting severity to `trace` logs all entry points enter and exits, this has a severe effect on performance. All other log levels will be interpreted as `none`.
 - `DXVK_NVAPI_LOG_PATH` enables file logging additionally to console output and sets the path where the log file `nvapi.log`/`nvapi64.log`/`nvofapi64.log` should be written to. Log statements are appended to an existing file. Please remove this file once in a while to prevent excessive grow. This requires `DXVK_NVAPI_LOG_LEVEL` set to `info` or `trace`.
 
+The following environment variables tweak DXVK-NVAPI's Vulkan Reflex layer's runtime behavior:
+
+- `DXVK_NVAPI_VKREFLEX=1` enables the layer
```

Comment:
For safety, even if this is mentioned earlier:

```
.. ` , when not set, the layer is disabled.`
```

Reply:
Sure, will do.